### ISSUE TITLE
[BUG]  Warning: Atom.jl: unrecognised message outline since last julia-client update

### ISSUE THREAD

**Issue by HenriDeh** [JunoLab/Juno.jl#350]
## Details
- Atom version: 1.40.1
- Julia version: 1.2
- OS: Linux Mint 19.2
- Package versions:
  - Atom.jl: 0.9.1 
  - julia-client: 0.10.1
  - ink: 0.11.0
  - Juno : 0.7.2

### Steps to reproduce

1. Type anything in a file opened in Atom and the Warning shows up (every key stroke):
```
┌ Warning: Atom.jl: unrecognised message outline.
└ @ Atom ~/.julia/packages/Atom/Wouyw/src/comm.jl:171
```
2. Happens only in .jl files

**Comment by pfitzseb** [comment]
You need to be on Atom.jl version 0.10 for this to work properly.

**Comment by tfburns** [comment]
Where can `Atom.jl` be found? I have the same issue.

**Comment by HenriDeh** [comment]
It's a Julia package, simply use Pkg.update("Atom").
________________________________

**Comment by ilgeary** [comment]
I fixed this issue once by updating to Julia 1.2.0 as described above.  But I updated to Cataline on my Mac this morning, and now this problem is back.  Any other suggestions???
